src/UI/Art/SpriteArtist/CommanderOverlayArtist.java

```diff
@@ -156,7 +156,9 @@ public static BufferedImage buildCoPowerBar(Commander co, int[] abilityPoints, i
     if( 0 == abilityPoints.length )
       return SpriteLibrary.createTransparentSprite(1, 1);
 
-    final double pixelsPerPowerUnit = 3.0 / Commander.CHARGERATIO_FUNDS;
+    // Just grab the first ability's star cost since I don't really want to consider what to do under variant star costs.
+    int starCost = co.myAbilities.get(0).costBasis.baseStarRatio;
+    final double pixelsPerPowerUnit = 3.0 / starCost;
```

Comment:
Remember when `pixelsPerPowerUnit` was just `3`? I remember.

It might be more clear/less magical to have `pixelsPerStar(or star equivalent)=3` and also `pixelsPerPowerPlanck=pixelsPerStar/starCost`. Or like, something, I dunno. The now-magic 3 just makes me double-take every time I see it now.